mdal/frmts/mdal_3di.hpp

```diff
@@ -51,7 +51,8 @@ namespace MDAL
       std::string getTimeVariableName() const override;
       std::set<std::string> ignoreNetCDFVariables() override;
       void parseNetCDFVariableMetadata( int varid, const std::string &variableName,
-                                        std::string &name, bool *is_vector, bool *is_x ) override;
+                                        std::string &name, bool *is_vector, bool *is_x,
+                                        std::map<std::string, std::string> &option ) override;
```

Comment:
option -> options

Reply:
Wouldn't be possible to use directly `Metadata` class https://github.com/lutraconsulting/MDAL/blob/417898347448e1dab7decc2d31a9c418873f9513/mdal/mdal_data_model.hpp#L40